magma/bit.py

```diff
@@ -179,20 +179,8 @@ def __int__(self) -> int:
         raise NotImplementedError("Converting magma bit to int not supported")
 
     def __repr__(self):
-        if self is VCC:
-            return "VCC"
-        if self is GND:
-            return "GND"
         return super().__repr__()
 
-
-VCC = Bit[Direction.Out](name="VCC")
-GND = Bit[Direction.Out](name="GND")
-
-HIGH = VCC
-LOW = GND
-
-
```

Comment:
Should we keep HIGH/LOW defined in this namespace for backwards compatibility? Probably not a major issue though

Reply:
Doen't see either of these used outside of digital.py...will keep there for now for backward compat. but can be removed.